[2024-01-05T22:01:23.378+0300] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Google_X_mention_score.train_model_exp scheduled__2024-01-03T02:30:00+00:00 [queued]>
[2024-01-05T22:01:23.389+0300] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Google_X_mention_score.train_model_exp scheduled__2024-01-03T02:30:00+00:00 [queued]>
[2024-01-05T22:01:23.389+0300] {taskinstance.py:1288} INFO - 
--------------------------------------------------------------------------------
[2024-01-05T22:01:23.390+0300] {taskinstance.py:1289} INFO - Starting attempt 1 of 2
[2024-01-05T22:01:23.390+0300] {taskinstance.py:1290} INFO - 
--------------------------------------------------------------------------------
[2024-01-05T22:01:23.400+0300] {taskinstance.py:1309} INFO - Executing <Task(BashOperator): train_model_exp> on 2024-01-03 02:30:00+00:00
[2024-01-05T22:01:23.403+0300] {standard_task_runner.py:55} INFO - Started process 181737 to run task
[2024-01-05T22:01:23.410+0300] {standard_task_runner.py:82} INFO - Running: ['airflow', 'tasks', 'run', 'Google_X_mention_score', 'train_model_exp', 'scheduled__2024-01-03T02:30:00+00:00', '--job-id', '458', '--raw', '--subdir', 'DAGS_FOLDER/google_mention_score.py', '--cfg-path', '/tmp/tmpqn2f9ogh']
[2024-01-05T22:01:23.412+0300] {standard_task_runner.py:83} INFO - Job 458: Subtask train_model_exp
[2024-01-05T22:01:23.521+0300] {task_command.py:389} INFO - Running <TaskInstance: Google_X_mention_score.train_model_exp scheduled__2024-01-03T02:30:00+00:00 [running]> on host prod-srv
[2024-01-05T22:01:23.645+0300] {taskinstance.py:1516} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=admin
AIRFLOW_CTX_DAG_ID=Google_X_mention_score
AIRFLOW_CTX_TASK_ID=train_model_exp
AIRFLOW_CTX_EXECUTION_DATE=2024-01-03T02:30:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-01-03T02:30:00+00:00
[2024-01-05T22:01:23.646+0300] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2024-01-05T22:01:23.647+0300] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python3 /home/igor/mlops_4/scripts/train_model.py exp']
[2024-01-05T22:01:23.653+0300] {subprocess.py:86} INFO - Output:
[2024-01-05T22:01:30.041+0300] {subprocess.py:93} INFO - Registered model 'exp_model' already exists. Creating a new version of this model...
[2024-01-05T22:01:30.141+0300] {subprocess.py:93} INFO - Created version '51' of model 'exp_model'.
[2024-01-05T22:01:30.575+0300] {subprocess.py:97} INFO - Command exited with return code 0
[2024-01-05T22:01:30.633+0300] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=Google_X_mention_score, task_id=train_model_exp, execution_date=20240103T023000, start_date=20240105T190123, end_date=20240105T190130
[2024-01-05T22:01:30.651+0300] {local_task_job.py:212} INFO - Task exited with return code 0
[2024-01-05T22:01:30.673+0300] {taskinstance.py:2596} INFO - 0 downstream tasks scheduled from follow-on schedule check
